models/vrp.rb

```diff
@@ -334,15 +334,14 @@ def self.convert_relation_lapse_into_lapses(hash)
         if Models::Relation::ONE_LAPSE_TYPES.include?(relation[:type])
           if relation[:lapse]
             relation[:lapses] = [relation[:lapse]]
-            relation.delete(:lapse)
           end
         elsif [:vehicle_trips, Models::Relation::SEVERAL_LAPSE_TYPES].flatten.include?(relation[:type])
```

Comment:
Not introduced in this PR but `Models::Relation::SEVERAL_LAPSE_TYPES` already includes `vehicle_trips` so the if condition can be simplified to be `Models::Relation::SEVERAL_LAPSE_TYPES.include?(relation[:type])`

Reply:
It seems it already belong to `Models::Relation::SEVERAL_LAPSE_TYPES` :open_mouth:

Reply:
Yeah, it was a change introduced with a PR comment, apparently we missed to correct this. I believe there is one more such location where we have `[:vehicle_trips, Models::Relation::SEVERAL_LAPSE_TYPES].flatten`